landoapi/decorators.py

```diff
@@ -23,12 +23,13 @@ class require_phabricator_api_key:
     will be used. If an API key is provided it will still be verified.
 
     Decorated functions may assume X-Phabricator-API-Key header is present,
-    contains a valid phabricator API key and flask.g.phabricator is a
+    contains a valid phabricator API key and the first argument is a
     PhabricatorClient using this API Key.
```

Comment:
Need to update the docstring to reflect that the this is based on `provide_client`.